src/libstd/fs.rs

```diff
@@ -422,6 +422,11 @@ impl OpenOptions {
     /// This option, when true, means that writes will append to a file instead
     /// of overwriting previous contents.
     ///
+    /// Note that this isn't the same as `write` with the cursor starting at
+    /// the end of the file; this mode only allows writing to the end of the
```

Comment:
Hm, I thought that this was basically the same as a `write` with the cursor starting at the end of the file? According to the manpage at least:

>               The  file  is opened in append mode.  Before each write(2), the file
              offset is positioned at the end of the file, as  if  with  lseek(2).
              O_APPEND may lead to corrupted files on NFS filesystems if more than
              one process appends data to a file at once.   This  is  because  NFS
              does  not  support  appending to a file, so the client kernel has to
              simulate it, which can't be done without a race condition.